test/economic-resource/resource_logic.js

```diff
@@ -20,6 +20,17 @@ const testEventProps = {
 runner.registerScenario('EconomicResource & EconomicEvent record interactions', async (s, t) => {
   const { cells: [observation, specification] } = await buildPlayer(s, config, ['observation', 'specification'])
 
+  // HACK: this prevents a flaky issue
```

Comment:
Have you reinstalled modules since encountering this issue? After bfde90bd we should no longer be hitting these intermittent timeouts. Unless you have a *really* slow machine... I guess increasing the timeout again is a possibility. But I would try ensuring you have the patch active and see if it resolves things.

Reply:
It's not a timeout, it seems like a race condition.

Reply:
Ah, it manifested in an unexpected way. You're right that those things were related. It didn't seem to me like they could be. I would remove these again